.gitignore

```diff
@@ -8,3 +8,4 @@
 *.*proj.user
 *.DotSettings.user
 *.userprefs
+/_ReSharper.Caches
```

Comment:
Odd, I've never seen R# create that folder. But then I'm mostly using Rider, so this might be something only the VS plugin does.

Reply:
Yeah, I have my ReSharper configured that way so it gets cleaned up regularly.